### ISSUE TITLE
Error while creating project

### ISSUE THREAD

**Issue by shikharish** [LoadingByte/cinecred#8]
I am facing this issue while creating a project. 
This is the error:
```
WARNING: Using incubator modules: jdk.incubator.foreign
WARNING: package com.apple.eawt not in java.desktop
6907 [AWT-EventQueue-0] SEVERE Cinecred - Uncaught exception. Will terminate the program.
java.lang.ExceptionInInitializerError
	at com.loadingbyte.cinecred.ui.helper.FontChooserWidget.populateFamilyComboBox(FormWidgets.kt:1196)
	at com.loadingbyte.cinecred.ui.helper.FontChooserWidget.<init>(FormWidgets.kt:1187)
	at com.loadingbyte.cinecred.ui.styling.StyleForm.makeBackingSettingWidget(StyleForm.kt:260)
	at com.loadingbyte.cinecred.ui.styling.StyleForm.makeSettingWidget(StyleForm.kt:96)
	at com.loadingbyte.cinecred.ui.styling.StyleForm.addSingleSettingWidget(StyleForm.kt:47)
	at com.loadingbyte.cinecred.ui.styling.StyleForm.<init>(StyleForm.kt:39)
	at com.loadingbyte.cinecred.ui.styling.StyleForm.<init>(StyleForm.kt:13)
	at com.loadingbyte.cinecred.ui.styling.EditStylingPanel.<init>(EditStylingPanel.kt:40)
	at com.loadingbyte.cinecred.ui.StylingDialog.<init>(ProjectDialogs.kt:36)
	at com.loadingbyte.cinecred.ui.ProjectController.<init>(ProjectController.kt:63)
	at com.loadingbyte.cinecred.ui.ctrl.MasterCtrl.openProject(MasterCtrl.kt:70)
	at com.loadingbyte.cinecred.ui.ctrl.WelcomeCtrl.tryOpenProject(WelcomeCtrl.kt:173)
	at com.loadingbyte.cinecred.ui.ctrl.WelcomeCtrl.projects_createConfigure_onClickDone$lambda$11$lambda$7(WelcomeCtrl.kt:403)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(Unknown Source)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.desktop/java.awt.EventQueue$4.run(Unknown Source)
	at java.desktop/java.awt.EventQueue$4.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.desktop/java.awt.EventQueue.dispatchEvent(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.run(Unknown Source)
Caused by: java.lang.IllegalStateException: Mysterious locale bn_IN in font java.awt.Font[family=Mitra ,name=Mitra Mono,style=plain,size=1].
	at com.loadingbyte.cinecred.ui.helper.FontSorter.getLocalizedSubfamilies(Fonts.kt:347)
	at com.loadingbyte.cinecred.ui.helper.FontSorter.sort(Fonts.kt:163)
	at com.loadingbyte.cinecred.ui.helper.FontFamilies.<init>(Fonts.kt:78)
	at com.loadingbyte.cinecred.ui.helper.FontsKt.<clinit>(Fonts.kt:16)
	... 26 more

```

OS: Linux Mint 21.1 x86_64
Cinecred version: 1.5.1

I believe the issue is with Java but could not fix it.
Java version:
openjdk 18.0.2-ea 2022-07-19
OpenJDK Runtime Environment (build 18.0.2-ea+9-Ubuntu-222.04)
OpenJDK 64-Bit Server VM (build 18.0.2-ea+9-Ubuntu-222.04, mixed mode, sharing)

**Comment by LoadingByte** [comment]
Thanks for the report! Cinecred got confused by the font family »Mitra«, which is installed on your system, and crashed. I've fixed this for the next version, but until then, I fear the only way to get around the crash is to uninstall the »Mitra« family from your system.

**Comment by shikharish** [comment]
Thank you, that worked!

**Comment by digitalextremist** [comment]
Looks like the same thing occurs with `Mitra Mono` and going to attempt same solution //

**Comment by digitalextremist** [comment]
Same here: Same font, same story. Same fix!

**Comment by LoadingByte** [comment]
Fixed in 1.6.0.
